getting-started/using-telerik-justmock-in-your-test-project.md

```diff
@@ -23,10 +23,10 @@ This topic demonstrates how to use __Telerik® JustMock__ in a new test project
 	**Figure 1: Add new project**  
 	![Add New Project to VS Solution](images/AddNewProject.png)
 
-1. Select the Test Project template. Enter a Name and click OK.
+1. Select the Test Project template or search for it by typing "JustMock test". After that select the Test Project template and click OK.

```

Comment:
```suggestion
1. Select the desired **JustMock Test Project** template from the menu and click OK. 
```
The first sentence instructs the reader to select the Test Project template. The second sentence duplicates the same instructions and asks the reader to  select the Test Project template again.
We can omit the template search instruction, I am not sure if the add more value.